### PR TITLE
Fix regression in `composer robo validate:drupal-config`

### LINKED ISSUE #75: Regression in `composer robo validate:drupal-config`
## Description
#64 Introduced a regression in `composer robo validate:drupal-config` causing perfectly fine config to return a false positive (true negative?):

Tugboat log snippet from CHQ.com:

```
2022-04-12T08:11:47.819Z - 62550b647a1d6289540bab66# /bin/sh -c composer robo validate:drupal-config
> Composer\Config::disableProcessTimeout
> robo --ansi 'validate:drupal-config'
 [Exec] Running /var/lib/tugboat/web/../vendor/bin/drush config:status --format=json in /var/lib/tugboat/web/sites/default/
 [Exec] Done in 2.159s
➜  []
 [error]    in task Usher\Robo\Plugin\Commands\ValidateConfigCommands 

  Drupal database configuration does not match the tracked file system configuration. 
Script robo --ansi handling the robo event returned with error code 1
2022-04-12T08:11:50.782Z - Command Failed (Tugboat Error 1064): Exit code (1)
```

For bonus sanity I've added a "bare" `drush config:status --format=json` (so without any Robo/Usher magic) which returns OK:

```
2022-04-12T08:11:45.579Z - 62550b647a1d6289540bab66# /bin/sh -c vendor/bin/drush config:status --format=json
 [notice] No differences between DB and sync directory.
[]
2022-04-12T08:11:47.819Z - 62550b647a1d6289540bab66# /bin/sh -c composer robo validate:drupal-config
> Composer\Config::disableProcessTimeout
> robo --ansi 'validate:drupal-config'
 [Exec] Running /var/lib/tugboat/web/../vendor/bin/drush config:status --format=json in /var/lib/tugboat/web/sites/default/
 [Exec] Done in 2.159s
➜  []
 [error]    in task Usher\Robo\Plugin\Commands\ValidateConfigCommands 

  Drupal database configuration does not match the tracked file system configuration. 
Script robo --ansi handling the robo event returned with error code 1
2022-04-12T08:11:50.782Z - Command Failed (Tugboat Error 1064): Exit code (1)
```
(full log [here](https://dashboard.tugboat.qa/log/62550b63d9027707d373f210)

### Steps
1. On a site using `chromatic/usher`, bump it from 2.15.0 to 2.16.0.
2. Deploy on Tugboat, making sure the `composer robo validate:drupal-config` command is used during the deploy.
3. Tugboat Deploy Log: Read 'em and weep.

### Environment
- Operating system and version: Tugboat

## Expected Behavior
Happy Usher, Happy life, Deployed site.

PR incoming

### PR DESCRIPTION
**Pull request by Boegie**
## Description
See code change comments and https://devrant.com/rants/764332/tfw-you-learn-that-trim-null-equals-empty-string-in-php-i-m-starting-to-understa

## Motivation / Context
#75 

## Testing Instructions / How This Has Been Tested
Not tested, testing Usher is a bit of a &*^*#@#! TBH. Seems to involve requiring a PR in a real project to show if it works.

### CODE REVIEW

File: src/Robo/Plugin/Commands/ValidateConfigCommands.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function validateDrupalConfig($siteDir = 'default'): void
             ->run();
         $drushOutput = trim($result->getOutputData());
```
Comment (Boegie) [inline src/Robo/Plugin/Commands/ValidateConfigCommands.php:51]: `$result->getOutputData()` returns `null|string`, trimming a `null`, so when there's no output, which means config is OK, returns an empty string...
_#WeAllLovePHP_

File: src/Robo/Plugin/Commands/ValidateConfigCommands.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ public function validateDrupalConfig($siteDir = 'default'): void
...
         $configJson = json_decode($drushOutput);
-        if (!is_null($configJson)) {
```
Comment (Boegie) [inline src/Robo/Plugin/Commands/ValidateConfigCommands.php:53]: ...and an empty string is not `null`, making this throwing the "We're-All-Gonna-Die-WTFBBQ!!!!!" `TaskException` when in fact all is well.

**Review by markdorison — APPROVED** [review]

**Comment by markdorison** [comment]
I published a new release with this fix.

**Comment by markdorison** [comment]
Nice catch and thanks for the fix @Boegie! 💯

**Comment by adamzimmermann** [comment]
Thank you for finding and fixing this before I even started my day! 🎉

**Pull request merged**